logic.js

```diff
@@ -0,0 +1,124 @@
+const Tweeter = () => {
+
+    let ind = 0;
+    let c = [];
```

Comment:
You should use clear and understandable names for classes, functions, and variables (for example: firstName etc.) and not something general like "arr", "str" etc.